doc/api/assert.md

```diff
@@ -256,19 +256,20 @@ If the values are not equal, an `AssertionError` is thrown with a `message`
 property set equal to the value of the `message` parameter. If the `message`
 parameter is undefined, a default error message is assigned.
 
-## assert.fail(message)
+## assert.fail([message])
 ## assert.fail(actual, expected, message, operator)
 <!-- YAML
 added: v0.1.21
 -->
 * `actual` {any}
 * `expected` {any}
-* `message` {any}
+* `message` {any} (default: 'Failed')
 * `operator` {string} (default: '!=')
 
 Throws an `AssertionError`. If `message` is falsy, the error message is set as
 the values of `actual` and `expected` separated by the provided `operator`.
 Otherwise, the error message is the value of `message`.
+If you provide no arguments at all, a default message will be used instead.
```

Comment:
I asked for this [out loud](https://twitter.com/addaleax/status/869678148367130627) on Twitter, and a protected account gave the answer of “good documentation informs the reader rather than instructing them”, which actually makes a lot of sense to me. It’s not like all usage of “you” violates that rule, but that it will tend to so seems reasonable.